xesmf/tests/test_frontend.py

```diff
@@ -713,6 +736,27 @@ def test_spatial_averager(poly, exp):
     assert 'my_geom' in out.dims
 
 
+@pytest.mark.xfail
+def test_spatial_averager_with_zonal_region():
```

Comment:
Should this be in this PR ? Looks related with your other work on the large regions issue.